README.md

```diff
@@ -15,18 +16,32 @@ This website was written by Godmode from the Eden Server with inspiration from F
 ## Running
 
 ### Production
+
 1. Install the client by changing to the `/client` directory and running `npm i`;
 2. Build running `npm run build` in the same directory.
 3. Navigate back to the root directory and run the `npm start` command on the shell.
 
 ### Development
+
 1. Install the client by changing to the `/client` directory and running `npm i`;
-1. Navigate back to the root directory and run  the `npm run dev` command on the shell.
+2. Navigate back to the root directory and run the `npm run dev` command on the shell.
+
+### Linting & Code Formatting
+
+The 'client' React app is setup with its own ESLint and Prettier config, plus a VSCode workspace config which enables format on save and issue highlighting. For this to work properly, open the 'client' folder as your project in VSCode. Test this is working by creating an issue such as incorrect tab spacing (Prettier), or referencing state or props directly in JSX to trigger a 'use destructuring' prompt (ESLint). You should see squiggly lines in the code and be able to hover over this for more info and potential quick fix options. Saving the file will solve any automatically fixable issues.
+
+### Notes on the 'client' app
+
+Ideally the 'client' React app should be able to function on its own without at the API. Therefore it should handle its own dependencies in terms of node modules and any configuration. Please ensure that any npm packages required for 'client' are installed at the 'client' level and not at the root API level.
```

Comment:
Good note addition for future devs. Server and client could be split into two repositories as well. I just did this for convenience.